src/platforms/ios/iaphandler.h

```diff
@@ -91,16 +97,20 @@ class IAPHandler final : public QAbstractListModel {
   struct Product {
     QString m_name;
     QString m_price;
+    QString m_monthlyPrice;
+    // This is not exposed and it's not localized. It's used to compute the
+    // saving %.
+    double m_nonLocalizedMonthlyPrice = 0;
```

Comment:
I would not store monetary values in `double` variables. I would store this as cent amount as an `unsigned int`. Even though we only use this for calculation for now we would then have an accurate price representation which might be handy.

Reply:
it must be double because often the price is 4.99. unsigned int is not enough. Note that localized monthly price is used only to compute the saving %.